src/devices/MAX31856/MAX31856.cs

```diff
@@ -0,0 +1,211 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Spi;
+using System.IO;
+using UnitsNet;
+
+namespace Iot.Device.MAX31856
+{
+    /// <summary>
+    /// Add documentation here
+    /// </summary>
+    public class MAX31856 : IDisposable
+    {
+        private readonly byte _thermocoupleType;
+        private SpiDevice _spiDevice;
+
+        #region SpiSettings
+
+        /// <summary>
+        /// Spi Clock Frequency from the Technical Manual of the device
+        /// </summary>
+        public const int SpiClockFrequency = 5_000_000;
+
+        /// <summary>
+        /// Spi Clock Frequency from the Technical Manual of the device
+        /// </summary>
+        public const int DataFlow = 0;
+
+        /// <summary>
+        /// MAX31856 SPI Mode
+        /// </summary>
+        public const SpiMode SpiMode = System.Device.Spi.SpiMode.Mode1;
+
+        #endregion
+
+        /// <summary>
+        /// Command to Get Temperature from the device
+        /// </summary>
+        /// <remarks>
+        /// Initializes the device and then reads the data next it converts the bytes to thermocouple temperature
+        /// </remarks>
+        public bool TryGetTemperature(out Temperature temperature)
+        {
+            try
+            {
+                temperature = ReadTemperature(out byte[] spiOutputData);
+                return true;
+            }
+            catch (IOException)
+            {
+                // The tempearture is set to 1 million if there is an exception
+                temperature = new Temperature();
+                // Write error message to council
+                return false;
+            }
+        }
+
+        /// <summary>
+        ///  Reads the temperature from the Cold-Junction sensor
+        /// </summary>
+        /// <returns>
+        ///  Temperature in degrees celsius
```

Comment:
It's just a temperature now :-) But you can add the precision and range. So would be something like (adjust to the real values):
/// Temperature, precision +- 0.01 Celsius range from -100 Celsius to +200 Celsius